### PR TITLE
Modified README to include steps for New Release

### PR DESCRIPTION
**Pull request by npecka**
Included a breakdown of steps to create a new release of DVO in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,62 @@ make test
...
+## Releases
+
+### New Release
+
+**Before Proceeding:** 
+* Assure desired changes for new release have been submitted and merged successfully
+* Check with team to verify if this is a MAJOR, MINOR, or a PATCH release
+
+**Release Process:**
+1. Create a new release
```
Comment (bkez322) [inline README.md:114]:
```suggestion
1. Create a new DVO release in GitHub
```

File: README.md [hunk 2]
```diff
@@ -101,6 +101,62 @@ make test
...
+* Check with team to verify if this is a MAJOR, MINOR, or a PATCH release
+
+**Release Process:**
+1. Create a new release 
+    
+    - Create new release on GitHub page from the right column
+    - Follow the model of Major/Minor/Patch (x/y/z, 0.1.2) 
+    - Provide a description of the release (Auto-generate or Manually)
+
+2. Publish the new release
```
Comment (bkez322) [inline README.md:120]:
```suggestion
2. Publish the new DVO release to Quay
```

File: README.md [hunk 3]
```diff
@@ -101,6 +101,62 @@ make test
...
+    - Create new release on GitHub page from the right column
+    - Follow the model of Major/Minor/Patch (x/y/z, 0.1.2) 
+    - Provide a description of the release (Auto-generate or Manually)
+
+2. Publish the new release 
+    
+    - This will trigger a jenkins job that will build a new image of DVO with the new release tag
+    - Verify Jenkins Job was successful - [DVO Jenkins](https://ci.int.devshift.net/view/deployment-validation-operator/job/app-sre-deployment-validation-operator-gh-build-tag/)
+
+3. Modify OLM
```
Comment (bkez322) [inline README.md:125]:
```suggestion
3. Publish new DVO release operator-hub
```

File: README.md [hunk 4]
```diff
@@ -101,6 +101,62 @@ make test
...
+
+    # Modify the following line to reflect the previous release version for upgrade purposes 
+    # (ex. If going from 0.2.1 -> 0.2.2, then the previous release was 0.2.1)
+
+    * spec.replaces: deployment-validation-operator.v<PREVIOUS RELEASE VERSION>
+    ```
+
+    - Submit a PR
+
+4. Verify new release changes by deploying and testing DVO
```
Comment (bkez322) [inline README.md:157]:
```suggestion
4. OLM updates DVO version across DVO-consuming kubernetes clusters (no action required)
```
(Right now DVO is in an alpha-state, and so clusters running an OLM that is configured to ignore alpha releases in Operator-Hub may have unreliable success with the following):

Once the merge request to the `k8s-operatorhub/community-operators` GitHub repo is merged, the latest version of DVO available through the Operator-Hub ecosystem should automatically update. You can check the latest version available [here](https://operatorhub.io/operator/deployment-validation-operator).

**Review by bkez322 — CHANGES_REQUESTED** [review]
I made a few suggestions to keep things abundantly clear and accurate. Otherwise, thanks for adding this!

File: README.md [hunk 5]
```diff
@@ -101,6 +101,62 @@ make test
...
+**Release Process:**
+1. Create a new release 
+    
+    - Create new release on GitHub page from the right column
+    - Follow the model of Major/Minor/Patch (x/y/z, 0.1.2) 
+    - Provide a description of the release (Auto-generate or Manually)
+
+2. Publish the new release 
+    
+    - This will trigger a jenkins job that will build a new image of DVO with the new release tag
```
Comment (bkez322) [inline README.md:122]:
```suggestion
    - Generating a new tagged release in GitHub will trigger a jenkins job that will build a new image of DVO with the new release tag
```

File: README.md [hunk 6]
```diff
@@ -101,6 +101,62 @@ make test
...
+    - OperatorHub Repository for DVO - [DVO OLM](https://github.com/k8s-operatorhub/community-operators/tree/main/operators/deployment-validation-operator)
+    - Edit deployment-validation-operator.package.yaml to reflect the new release
+    
+    ```yaml
+    # REALEASE VERSION == 0.2.0, 0.2.1, etc.
+    
+    * channels.currentCSV: deployment-validation-operator.v<RELEASE VERSION>
+    ```
+    
+    - Clone a new directory from a previous version (ex. 0.2.0, 0.2.1) and change the name of the directory to reflect the new release
```
Comment (bkez322) [inline README.md:136]:
```suggestion
    - As a shortcut, you may choose to copy-+paste the most recent already-existing DVO version directory (ex. 0.2.0, 0.2.1) and change the name of the directory to reflect the new release version
```

File: README.md [hunk 7]
```diff
@@ -101,6 +101,62 @@ make test
...
+    - Modify the clusterserviceversion file's name within the directory to reflect the new release version
+    
+    ```yaml
+    # Edit the clusterserviceversion file within the directory and modify the following lines to reflect the new release
+    # REALEASE VERSION == 0.2.0, 0.2.1, etc.
```
Comment (bkez322) [inline README.md:141]:
```suggestion
    # RELEASE VERSION == 0.2.0, 0.2.1, etc.
```
typo

File: README.md [hunk 8]
```diff
@@ -101,6 +101,62 @@ make test
...
+    * spec.links.url: https://quay.io/deployment-validation-operator/dv-operator:<RELEASE VERSION>
+    * spec.version: <RELEASE VERSION>
+
+    # Modify the following line to reflect the previous release version for upgrade purposes 
+    # (ex. If going from 0.2.1 -> 0.2.2, then the previous release was 0.2.1)
+
+    * spec.replaces: deployment-validation-operator.v<PREVIOUS RELEASE VERSION>
+    ```
+
+    - Submit a PR
```
Comment (bkez322) [inline README.md:155]: Could we also mention the reviewers file and what it does for the users in it?

File: README.md [hunk 9]
```diff
@@ -101,6 +101,62 @@ make test
...
+    * spec.install.spec.deployments.spec.template.spec.containers.image: quay.io/deployment-validation-operator/dv-operator:<RELEASE VERSION>
+    * spec.links.url: https://quay.io/deployment-validation-operator/dv-operator:<RELEASE VERSION>
+    * spec.version: <RELEASE VERSION>
+
+    # Modify the following line to reflect the previous release version for upgrade purposes 
+    # (ex. If going from 0.2.1 -> 0.2.2, then the previous release was 0.2.1)
+
+    * spec.replaces: deployment-validation-operator.v<PREVIOUS RELEASE VERSION>
+    ```
+
```
Comment (bkez322) [inline README.md:154]:
```suggestion
- If need-be for the nature of what the changes in the new DVO release, update the rest of these files accordingly
```

**Comment by bkez322** [comment]
Another though I had:

In the interest of keeping the scope of the single README file from becoming too big, I think I would like it if we could create a new `docs` directory in the DVO repository on the outermost level, and move all of these doc changes into a brand new doc in that directory.

And then we can have the README just link that doc.

What do you think?

**Comment by npecka** [comment]
> Another though I had:
> 
> In the interest of keeping the scope of the single README file from becoming too big, I think I would like it if we could create a new `docs` directory in the DVO repository on the outermost level, and move all of these doc changes into a brand new doc in that directory.
> 
> And then we can have the README just link that doc.
> 
> What do you think?

I think that would be great for organization. I can move this change and create a folder for that and get that process started with the releases change.

File: README.md (hunk 8 above)
Comment (npecka) [reply README.md]: I made a change for this. Can you double check this is how you want that

Comment (bkez322) [reply README.md]: Looks good, thanks!

**Pull request merged**